v2/e2e/coil_test.go

```diff
@@ -272,9 +280,25 @@ var _ = Describe("Coil", func() {
 			}
 			return int(depl.Status.ReadyReplicas)
 		}).Should(Equal(2))
+
+		By("deleting Egress")
+		kubectlSafe(nil, "delete", "-f", "manifests/egress.yaml")
```

Comment:
Now `manifests/egress.yaml` includes internet Namespace, so deleting the namespace should trigger the PDB deletion.
I would like to confirm the PDB to be deleted without deleting the entire Namespace. Is it possible?

Reply:
https://github.com/cybozu-go/coil/pull/275/commits/0e44d8d83cce0fa2ce797c0cf1d66854fe0f0bee